_docs/README.md

```diff
@@ -285,7 +294,8 @@ non-binary file through the [Go Template](https://golang.org/pkg/text/template)
 data structure.
 
 For example, if you had a variable called `Title` in your `boilerplate.yml` file, then you could access that variable
-in any of your templates using the syntax `{{"{{"}}.Title{{"}}"}}`.
+in any of your templates using the syntax `{{"{{"}}.Title{{"}}"}}`. You can also use Go template syntax to do
+if-statements, for loops, and use the provided [template helpers](#template-helpers).
```

Comment:
Not markdown, but boilerplate. That's because this README is, itself, processed by boilerplate as part of the CI job (note that it's in the `_docs` version, but the final version ends up in the root folder).